javascript/statements.json

```diff
@@ -1239,7 +1239,7 @@
         },
         "import_assertions": {
           "__compat": {
-            "description": "Import assertions",
+            "description": "Import attributes (`assert` syntax)",
```

Comment:
Not sure if we should rename this. If we do, maybe add "deprecated" for clarity?

```suggestion
            "description": "Import attributes (deprecated `assert` syntax)",
```

Reply:
I think that the original name would still be better, especially as the identifier does not change.

Reply:
And it's already marked as deprecated in `status.deprecated`.

Reply:
The proposal, which still includes the `assert` syntax, has been renamed to "import attributes". Also, even when using the deprecated syntax it does not have "assert" semantics anymore, but it has attribute semantics.

Reply:
Ah okay, well then as @queengooborg says we probably want to change the ID here in line 1240.

Reply:
Yes, I think that would work.